task 3.txt

```diff
@@ -0,0 +1,5 @@
+solution :
+-file for managing the features and users (csv or yml)
+-secret manager for storing the password and other needed info of the users
```

Comment:
Which secret manager? 
How would you integrate with it?
Who would have access and how would they consume it?
How would each user get their credentials?

Reply:
I thought about aws secret manager.
to integrate with it after I stored the data in aws secret manager- I will retrieve the secrets in my scripts using AWS CLI in PowerShell and assign appropriate permissions using IAM policies.
only relevant Admins/devops will get permission and also relevant resources like ec2 machines.
after the system creates the user on the server, the user can log in using the pre-configured password.